.eslintrc.js

```diff
@@ -69,5 +103,5 @@ module.exports = {
     },
   ],
 
-  ignorePatterns: ['!.eslintrc.js', '!.prettierrc.js', 'dist/'],
+  ignorePatterns: ['!.eslintrc.js', '!.prettierrc.js', 'dist/', 'docs/'],
```

Comment:
Good call, maybe we should do this for the module template.

Reply:
Looks like this is already done in the module template.

Reply:
Ah my bad. Cool :)